


<!DOCTYPE html>
<html class="writer-html5" lang="en" data-content_root="./">
<head>
  <meta charset="utf-8" /><meta name="viewport" content="width=device-width, initial-scale=1" />

  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
  <title>References &mdash; Cut&amp;Tag Analysis Pipeline 1.0.0 documentation</title>
      <link rel="stylesheet" type="text/css" href="_static/pygments.css?v=b86133f3" />
      <link rel="stylesheet" type="text/css" href="_static/css/theme.css?v=e59714d7" />
      <link rel="stylesheet" type="text/css" href="_static/copybutton.css?v=76b2166b" />
      <link rel="stylesheet" type="text/css" href="_static/css/custom.css?v=9482ece9" />

  
    <link rel="canonical" href="https://akhileshkaushal.github.io/cutntag/references.html" />
      <script src="_static/jquery.js?v=5d32c60e"></script>
      <script src="_static/_sphinx_javascript_frameworks_compat.js?v=2cd50e6c"></script>
      <script src="_static/documentation_options.js?v=4ebf8126"></script>
      <script src="_static/doctools.js?v=9bcbadda"></script>
      <script src="_static/sphinx_highlight.js?v=dc90522c"></script>
      <script src="_static/clipboard.min.js?v=a7894cd8"></script>
      <script src="_static/copybutton.js?v=f281be69"></script>
    <script src="_static/js/theme.js"></script>
    <link rel="index" title="Index" href="genindex.html" />
    <link rel="search" title="Search" href="search.html" />
    <link rel="prev" title="Case Study" href="casestudy.html" />
   
  <link rel="icon" href="_static/images/favicon.ico">

</head>

<body class="wy-body-for-nav"> 
  <div class="wy-grid-for-nav">
    <nav data-toggle="wy-nav-shift" class="wy-nav-side">
      <div class="wy-side-scroll">
        <div class="wy-side-nav-search" >

          
          
          <a href="index.html" class="icon icon-home">
            Cut&Tag Analysis Pipeline
          </a>
<div role="search">
  <form id="rtd-search-form" class="wy-form" action="search.html" method="get">
    <input type="text" name="q" placeholder="Search docs" aria-label="Search docs" />
    <input type="hidden" name="check_keywords" value="yes" />
    <input type="hidden" name="area" value="default" />
  </form>
</div>
        </div><div class="wy-menu wy-menu-vertical" data-spy="affix" role="navigation" aria-label="Navigation menu">
              <p class="caption" role="heading"><span class="caption-text">Pipeline Documentation</span></p>
<ul class="current">
<li class="toctree-l1"><a class="reference internal" href="intro.html">Cut&amp;Tag Preprocessing and Analysis Pipeline</a><ul>
<li class="toctree-l2"><a class="reference internal" href="intro.html#overview">Overview</a></li>
<li class="toctree-l2"><a class="reference internal" href="intro.html#key-applications">Key Applications</a></li>
<li class="toctree-l2"><a class="reference internal" href="intro.html#benefits-over-chip-seq">Benefits Over ChIP-seq</a></li>
<li class="toctree-l2"><a class="reference internal" href="intro.html#supported-experimental-designs">Supported Experimental Designs</a></li>
<li class="toctree-l2"><a class="reference internal" href="intro.html#typical-workflow-steps">Typical Workflow Steps</a></li>
</ul>
</li>
<li class="toctree-l1"><a class="reference internal" href="install.html">Installation</a><ul>
<li class="toctree-l2"><a class="reference internal" href="install.html#system-requirements">System Requirements</a></li>
<li class="toctree-l2"><a class="reference internal" href="install.html#installation-methods">Installation Methods</a></li>
<li class="toctree-l2"><a class="reference internal" href="install.html#optional-setup-for-gpu-enabled-servers-for-visualization-or-peak-modeling">Optional Setup for GPU-enabled Servers (for visualization or peak modeling)</a></li>
<li class="toctree-l2"><a class="reference internal" href="install.html#best-practices">Best Practices</a></li>
<li class="toctree-l2"><a class="reference internal" href="install.html#testing-your-installation">Testing Your Installation</a></li>
<li class="toctree-l2"><a class="reference internal" href="install.html#next-steps">Next Steps</a></li>
</ul>
</li>
<li class="toctree-l1"><a class="reference internal" href="run_pipeline.html">Cut&amp;Tag Analysis Pipeline: Step-by-Step</a><ul>
<li class="toctree-l2"><a class="reference internal" href="run_pipeline.html#overview-of-differences-from-chip-seq">Overview of Differences from ChIP-seq</a></li>
<li class="toctree-l2"><a class="reference internal" href="run_pipeline.html#pipeline-steps">Pipeline Steps</a></li>
<li class="toctree-l2"><a class="reference internal" href="run_pipeline.html#quality-control">1. Quality Control</a></li>
<li class="toctree-l2"><a class="reference internal" href="run_pipeline.html#adapter-trimming">2. Adapter Trimming</a></li>
<li class="toctree-l2"><a class="reference internal" href="run_pipeline.html#alignment-to-reference-genome">3. Alignment to Reference Genome</a></li>
<li class="toctree-l2"><a class="reference internal" href="run_pipeline.html#post-alignment-processing">4. Post-Alignment Processing</a></li>
<li class="toctree-l2"><a class="reference internal" href="run_pipeline.html#fragment-size-estimation">5. Fragment Size Estimation</a></li>
<li class="toctree-l2"><a class="reference internal" href="run_pipeline.html#peak-calling">6. Peak Calling</a></li>
<li class="toctree-l2"><a class="reference internal" href="run_pipeline.html#peak-annotation">7. Peak Annotation</a></li>
<li class="toctree-l2"><a class="reference internal" href="run_pipeline.html#signal-visualization">8. Signal Visualization</a></li>
<li class="toctree-l2"><a class="reference internal" href="run_pipeline.html#replicate-quality-assessment">9. Replicate Quality Assessment</a></li>
<li class="toctree-l2"><a class="reference internal" href="run_pipeline.html#downstream-analyses-optional">10. Downstream Analyses (Optional)</a></li>
</ul>
</li>
<li class="toctree-l1"><a class="reference internal" href="advanced_analysis.html">Advanced Analysis Options</a><ul>
<li class="toctree-l2"><a class="reference internal" href="advanced_analysis.html#spike-in-normalization-optional">1. Spike-in Normalization (Optional)</a></li>
<li class="toctree-l2"><a class="reference internal" href="advanced_analysis.html#read-deduplication">2. Read Deduplication</a></li>
<li class="toctree-l2"><a class="reference internal" href="advanced_analysis.html#background-signal-correction">3. Background Signal Correction</a></li>
<li class="toctree-l2"><a class="reference internal" href="advanced_analysis.html#comparative-peak-analysis">4. Comparative Peak Analysis</a></li>
<li class="toctree-l2"><a class="reference internal" href="advanced_analysis.html#fragmentation-pattern-analysis">5. Fragmentation Pattern Analysis</a></li>
</ul>
</li>
<li class="toctree-l1"><a class="reference internal" href="test_structure_containerisation.html">Test Dataset and Usage Guide</a><ul>
<li class="toctree-l2"><a class="reference internal" href="test_structure_containerisation.html#input-requirements">Input Requirements</a></li>
<li class="toctree-l2"><a class="reference internal" href="test_structure_containerisation.html#outputs">Outputs</a></li>
<li class="toctree-l2"><a class="reference internal" href="test_structure_containerisation.html#usage">Usage</a></li>
<li class="toctree-l2"><a class="reference internal" href="test_structure_containerisation.html#configuration">Configuration</a></li>
<li class="toctree-l2"><a class="reference internal" href="test_structure_containerisation.html#sample-metadata">Sample Metadata</a></li>
<li class="toctree-l2"><a class="reference internal" href="test_structure_containerisation.html#project-layout">Project Layout</a></li>
<li class="toctree-l2"><a class="reference internal" href="test_structure_containerisation.html#wrapper-script-run-cutandtag-pipeline-sh">Wrapper Script: <code class="docutils literal notranslate"><span class="pre">run_cutandtag_pipeline.sh</span></code></a></li>
<li class="toctree-l2"><a class="reference internal" href="test_structure_containerisation.html#container-support">Container Support</a></li>
</ul>
</li>
<li class="toctree-l1"><a class="reference internal" href="casestudy.html">Case Study</a><ul>
<li class="toctree-l2"><a class="reference internal" href="casestudy.html#case-study-single-cell-and-low-input-chromatin-profiling-using-cut-tag">Case Study: Single-Cell and Low-Input Chromatin Profiling Using CUT&amp;Tag</a></li>
<li class="toctree-l2"><a class="reference internal" href="casestudy.html#background-and-rationale">1. Background and Rationale</a></li>
<li class="toctree-l2"><a class="reference internal" href="casestudy.html#experimental-strategy-and-innovations">2. Experimental Strategy and Innovations</a></li>
<li class="toctree-l2"><a class="reference internal" href="casestudy.html#key-findings-and-results">3. Key Findings and Results</a></li>
<li class="toctree-l2"><a class="reference internal" href="casestudy.html#detailed-protocol-modifications">4. Detailed Protocol Modifications</a></li>
<li class="toctree-l2"><a class="reference internal" href="casestudy.html#data-analysis">5. Data Analysis</a></li>
<li class="toctree-l2"><a class="reference internal" href="casestudy.html#biological-insights">6. Biological Insights</a></li>
<li class="toctree-l2"><a class="reference internal" href="casestudy.html#future-applications">7. Future Applications</a></li>
</ul>
</li>
<li class="toctree-l1"><a class="reference internal" href="casestudy.html#conclusion">Conclusion</a></li>
<li class="toctree-l1 current"><a class="current reference internal" href="#">References</a><ul>
<li class="toctree-l2"><a class="reference internal" href="#key-publications">Key Publications</a></li>
</ul>
</li>
</ul>

        </div>
      </div>
    </nav>

    <section data-toggle="wy-nav-shift" class="wy-nav-content-wrap"><nav class="wy-nav-top" aria-label="Mobile navigation menu" >
          <i data-toggle="wy-nav-top" class="fa fa-bars"></i>
          <a href="index.html">Cut&Tag Analysis Pipeline</a>
      </nav>

      <div class="wy-nav-content">
        <div class="rst-content style-external-links">
          <div role="navigation" aria-label="Page navigation">
  <ul class="wy-breadcrumbs">
      <li><a href="index.html" class="icon icon-home" aria-label="Home"></a></li>
      <li class="breadcrumb-item active">References</li>
      <li class="wy-breadcrumbs-aside">
            <a href="_sources/references.rst.txt" rel="nofollow"> View page source</a>
      </li>
  </ul><div class="rst-breadcrumbs-buttons" role="navigation" aria-label="Sequential page navigation">
        <a href="casestudy.html" class="btn btn-neutral float-left" title="Case Study" accesskey="p"><span class="fa fa-arrow-circle-left" aria-hidden="true"></span> Previous</a>
  </div>
  <hr/>
</div>
          <div role="main" class="document" itemscope="itemscope" itemtype="http://schema.org/Article">
           <div itemprop="articleBody">
             
  <section id="references">
<span id="id1"></span><h1>References<a class="headerlink" href="#references" title="Link to this heading"></a></h1>
<section id="key-publications">
<h2>Key Publications<a class="headerlink" href="#key-publications" title="Link to this heading"></a></h2>
<ol class="arabic simple">
<li><p>Kaya-Okur, H. S., Wu, S. J., Codomo, C. A., Pledger, E. S., Bryson, T. D., Henikoff, J. G., Ahmad, K., &amp; Henikoff, S. (2019). CUT&amp;Tag for efficient epigenomic profiling of small samples and single cells. <em>Nature Communications</em>, 10, 1930. [<a class="reference external" href="https://doi.org/10.1038/s41467-019-09982-5](https://doi.org/10.1038/s41467-019-09982-5">https://doi.org/10.1038/s41467-019-09982-5](https://doi.org/10.1038/s41467-019-09982-5</a>)</p></li>
<li><p>Zhang, Y., Liu, T., Meyer, C. A., Eeckhoute, J., Johnson, D. S., Bernstein, B. E., … &amp; Liu, X. S. (2008). Model-based analysis of ChIP-Seq (MACS). <em>Genome Biology</em>, 9(9), R137. [<a class="reference external" href="https://doi.org/10.1186/gb-2008-9-9-r137](https://doi.org/10.1186/gb-2008-9-9-r137">https://doi.org/10.1186/gb-2008-9-9-r137](https://doi.org/10.1186/gb-2008-9-9-r137</a>)</p></li>
<li><p>Ramírez, F., Dündar, F., Diehl, S., Grüning, B. A., &amp; Manke, T. (2014). deepTools: a flexible platform for exploring deep-sequencing data. <em>Nucleic Acids Research</em>, 42(W1), W187–W191. [<a class="reference external" href="https://doi.org/10.1093/nar/gku365](https://doi.org/10.1093/nar/gku365">https://doi.org/10.1093/nar/gku365](https://doi.org/10.1093/nar/gku365</a>)</p></li>
<li><p>Yu, G., Wang, L. G., &amp; He, Q. Y. (2015). ChIPseeker: an R/Bioconductor package for ChIP peak annotation, comparison and visualization. <em>Bioinformatics</em>, 31(14), 2382–2383. [<a class="reference external" href="https://doi.org/10.1093/bioinformatics/btv145](https://doi.org/10.1093/bioinformatics/btv145">https://doi.org/10.1093/bioinformatics/btv145](https://doi.org/10.1093/bioinformatics/btv145</a>)</p></li>
<li><p>Quinlan, A. R., &amp; Hall, I. M. (2010). BEDTools: a flexible suite of utilities for comparing genomic features. <em>Bioinformatics</em>, 26(6), 841–842. [<a class="reference external" href="https://doi.org/10.1093/bioinformatics/btq033](https://doi.org/10.1093/bioinformatics/btq033">https://doi.org/10.1093/bioinformatics/btq033](https://doi.org/10.1093/bioinformatics/btq033</a>)</p></li>
<li><p>Dobin, A., Davis, C. A., Schlesinger, F., Drenkow, J., Zaleski, C., Jha, S., … &amp; Gingeras, T. R. (2013). STAR: ultrafast universal RNA-seq aligner. <em>Bioinformatics</em>, 29(1), 15–21. [<a class="reference external" href="https://doi.org/10.1093/bioinformatics/bts635](https://doi.org/10.1093/bioinformatics/bts635">https://doi.org/10.1093/bioinformatics/bts635](https://doi.org/10.1093/bioinformatics/bts635</a>)</p></li>
<li><p>Love, M. I., Huber, W., &amp; Anders, S. (2014). Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2. <em>Genome Biology</em>, 15(12), 550. [<a class="reference external" href="https://doi.org/10.1186/s13059-014-0550-8](https://doi.org/10.1186/s13059-014-0550-8">https://doi.org/10.1186/s13059-014-0550-8](https://doi.org/10.1186/s13059-014-0550-8</a>)</p></li>
<li><p>Yu, G., Wang, L. G., Han, Y., &amp; He, Q. Y. (2012). clusterProfiler: an R package for comparing biological themes among gene clusters. <em>OMICS: A Journal of Integrative Biology</em>, 16(5), 284–287. [<a class="reference external" href="https://doi.org/10.1089/omi.2011.0118](https://doi.org/10.1089/omi.2011.0118">https://doi.org/10.1089/omi.2011.0118](https://doi.org/10.1089/omi.2011.0118</a>)</p></li>
<li><p>Gaspar, J. M. (2018). NGMerge: merging paired-end reads via novel empirically-derived models of sequencing errors. <em>BMC Bioinformatics</em>, 19, 536. [<a class="reference external" href="https://doi.org/10.1186/s12859-018-2537-8](https://doi.org/10.1186/s12859-018-2537-8">https://doi.org/10.1186/s12859-018-2537-8](https://doi.org/10.1186/s12859-018-2537-8</a>)</p></li>
<li><p>Picard Toolkit. (2019). Broad Institute, GitHub Repository. [<a class="reference external" href="https://broadinstitute.github.io/picard/](https://broadinstitute.github.io/picard/">https://broadinstitute.github.io/picard/](https://broadinstitute.github.io/picard/</a>)</p></li>
</ol>
</section>
</section>


           </div>
          </div>
          <footer><div class="rst-footer-buttons" role="navigation" aria-label="Footer">
        <a href="casestudy.html" class="btn btn-neutral float-left" title="Case Study" accesskey="p" rel="prev"><span class="fa fa-arrow-circle-left" aria-hidden="true"></span> Previous</a>
    </div>

  <hr/>

  <div role="contentinfo">
    <p>&#169; Copyright 2025, Akhilesh Kaushal.</p>
  </div>

   

</footer>
        </div>
      </div>
    </section>
  </div>
  <script>
      jQuery(function () {
          SphinxRtdTheme.Navigation.enable(true);
      });
  </script> 

</body>
</html>